pkg/cmd/bootstrap.go

```diff
@@ -276,33 +277,34 @@ func setAccessToken(io *BootstrapParameters) error {
 	return nil
 }
 
-func checkBootstrapDependencies(io *BootstrapParameters, client *utility.Client, spinner status) error {
+func checkBootstrapDependencies(io *BootstrapParameters, client *utility.Client, spinner utility.Status) error {
 	missingDeps := []string{}
 	log.Progressf("\nChecking dependencies\n")
 
 	// in case custom Sealed Secrets namespace/service name are provided, try them first
 	// We do not add Sealed Secret Operator to missingDeps since we this dependency can be resolved
 	// by optional flags or interactive user inputs.
-	if (io.BootstrapOptions.SealedSecretsService.Namespace != "" && io.BootstrapOptions.SealedSecretsService.Namespace != defaultSealedSecretsServiceName.Namespace) ||
-		(io.BootstrapOptions.SealedSecretsService.Name != "" && (io.BootstrapOptions.SealedSecretsService.Name != defaultSealedSecretsServiceName.Name)) {
+	if !io.Insecure && ((io.BootstrapOptions.SealedSecretsService.Namespace != "" && io.BootstrapOptions.SealedSecretsService.Namespace != defaultSealedSecretsServiceName.Namespace) ||
+		(io.BootstrapOptions.SealedSecretsService.Name != "" && (io.BootstrapOptions.SealedSecretsService.Name != defaultSealedSecretsServiceName.Name))) {
 
 		spinner.Start("Checking if Sealed Secrets is installed with custom configuration", false)
 		if err := checkAndSetSealedSecretsConfig(io, client, io.BootstrapOptions.SealedSecretsService); err != nil {
-
 			warnIfNotFound(spinner, "Provided Sealed Secrets namespace/name are not valid. Please verify", err)
 			if !apierrors.IsNotFound(err) {
 				return fmt.Errorf("failed to check for Sealed Secrets Operator: %w", err)
 			}
 		}
 	} else {
-		// use default configuration to interact with Sealed Secrets
-
-		spinner.Start("Checking if Sealed Secrets is installed with the default configuration", false)
-		if err := checkAndSetSealedSecretsConfig(io, client, defaultSealedSecretsServiceName); err != nil {
-
-			warnIfNotFound(spinner, "Please install Sealed Secrets Operator from OperatorHub", err)
-			if !apierrors.IsNotFound(err) {
-				return fmt.Errorf("failed to check for Sealed Secrets Operator: %w", err)
+		if io.Insecure {
+			utility.DisplayUnsealedSecretsWarning()
+		} else {
+			spinner.Start("Checking if Sealed Secrets is installed with the default configuration", false)
+			if err := checkAndSetSealedSecretsConfig(io, client, defaultSealedSecretsServiceName); err != nil {
+				// use default configuration to interact with Sealed Secrets
+				warnIfNotFound(spinner, "The Sealed Secrets Operator was not detected", err)
```

Comment:
I'd keep the original message "Please install Sealed Secrets Operator from OperatorHub"

Reply:
Hi @wtam, I changed it because the Sealed Secrets Operator could be installed already but not in our expected default configuration.  Do you want me to change this or perhaps use something different?